contrib/slack.tpl

```diff
@@ -0,0 +1,49 @@
+{
+    "blocks": [
+        {
+            "type": "header",
+            "text": {
+                "type": "plain_text",
+                "text": ":rotating_light: VULNERABILITIES FOUND:\n{{ escapeXML ( index . 0 ).Target }}",
+                "emoji": true
+            }
+        },
+        {{- range . }}
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "*Scan type: {{ escapeXML .Type }}*",
+                }
+            },
+            {{- if (gt (len .Vulnerabilities) 40) }}
+                {
+                "type": "section",
+                    "text": {
+                    "type": "mrkdwn",
+                    "text": ":scream: *{{ (len .Vulnerabilities) | toString }} vulnerabilities found!* \nThis is too many for Slack to render!\nPlease <https://aquasecurity.github.io/trivy/v0.37/getting-started/installation/|install> & <https://aquasecurity.github.io/trivy/v0.37/docs/target/container_image/|run> trivy locally to see the full list."
```

Comment:
You can replace `v0.37` with `latest` to make it more dynamic.

Reply:
Nice 👍 
Thanks and done